db/migrate/20200404090436_remove_ga_cookies_preference.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveGaCookiesPreference < ActiveRecord::Migration
+  def up
+    Spree::Preference
```

Comment:
We should define that model within the migration to not depend on app code.

Reply:
:+1: done